keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -222,11 +225,10 @@ open class SqlResourceRepository(
 
   private fun doAppendHistory(event: PersistentEvent, ref: String) {
     log.debug("Appending event: $event")
-    jooq.transaction { config ->
```

Comment:
The new transaction here was unnecessary, and was breaking the expected transactional behavior of `CombinedRepository.upsertDeliveryConfig`.

Reply:
I went back and forth on this. It looks like executing those two queries outside of a transaction should be harmless, but it's always difficult to reason about such things.

Reply:
Yeah, I discussed this a bit with @robfletcher and it seemed like an OK approach (we do reads with follow-up writes outside transactions in other places).